### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by newhorizon-tech**
- Setup React and Redux 
- Setup project structure 
- Add React Router
- Add linters

### CODE REVIEW

File: src/pages/CategoriesPage.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const CategoriesPage = () => (
+  <h1> Categories </h1>
+);
+
+export default CategoriesPage;
```
Comment (dasileker) [inline src/pages/CategoriesPage.js]: - [x] The category should display under construction.

aaaa

File: src/pages/BooksPage.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const BooksPage = () => (
+  <h1> Books </h1>
+);
+
```
Comment (dasileker) [inline src/pages/BooksPage.js]: - [x] The book should have a form and a submit button.

rrrrrr

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @newhorizon-tech,** 

### **Nice Work 👍**

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- The **page** folder should be renamed **Component** and the **BooksPage** should contain a  prop data called from the bookList, I know it's not in your requirement but it will be just for displaying a list of books `title and author and category`.


ZZZ



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by newhorizon-tech** [comment]
- Refactored project using components 
- Added display book list functionality 
- Added a form for adding books

**Review by AshakaE — APPROVED** [review]
### Hi @newhorizon-tech  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**